lib/miq_expression/field.rb

```diff
@@ -31,6 +31,12 @@ def self.is_field?(field)
     !!(model < ApplicationRecord)
   end
 
+  def self.attribute?(val)
```

Comment:
can you add single comment line explaining the difference between `attribute?` and `is_field?`